### ISSUE TITLE
No changes detected when comparing a fresh image with a repo

### ISSUE THREAD

**Issue by Uko** [pharo-vcs/iceberg#293]
When I download the latest image, load Iceberg into it, add QualityAssistant repo and go to synchronize, there are no changes detected. However, when I add the same repo to MC via filetree (I add the directory where Iceberg cloned the repo), MC detects changes.

At the moment there are changes in the image which are missing in the git repo, so one can check the issue.

The repo is here: https://github.com/Uko/QualityAssistant

**Comment by Uko** [comment]
I think that the issue is in a non-dirty package.

If you create a new package, save it with MC, and add it in Iceberg, the synchronization diff will not detect any changes

**Comment by olekscode** [comment]
I also have this problem, and I really hope that it will be fixed.
Until then, here is a quick workaround for those who are reading this issue to find a solution:

Make some tiny change in any file of your package (add a space for example). It will allow Iceberg to see local changes and let you make a commit. Git will browse the diff and include your other changes to the package into your commit.

**Comment by Uko** [comment]
@olekscode do you mean putting a space in a method definition inside of the image, or changing the filetree file on the filesystem?

**Comment by olekscode** [comment]
@Uko in a method. The trick is to make some change in the code after committing with Monticello. This will mark the package as "dirty" AND make sure that Iceberg sees some local changes in it. Then (I think) git will compare the versions and include the changes you've made before your Monticello commit.

I have tried the trick you suggested with
`#Kernel asPackage mcPackage workingCopy modified: true`
But it results in an empty commit. Iceberg allows you to make it, but no changes are included

That's what happened to me when I tried. I don't know how it all works inside, so I may be mistaking
